container-registry/gcp/artifact-registry/variables.tf

```diff
@@ -1,5 +1,5 @@
 variable "kms_key_name" {
-  description = "KMS key name to encrypt GCP repositories"
+  description = "KMS key name to encrypt GCP repositories. Has the form: projects/{{my-project}}/locations/{{my-region}}/keyRings/{{my-kr}}/cryptoKeys/{{my-key}}"
```

Comment:
This looks like an id, not a name

Reply:
yes it is the id of the kms, do I have to update the name of the variable ?

Reply:
Yes you can update the name of the variable